dll/win32/msgina/lang/pt-PT.rc

```diff
@@ -0,0 +1,227 @@
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
```

Comment:
here it is portuguese, neutral for `pt-PT.rc`,
below is a file named just `pt.rc` that has portuguese, neutral.
I am not sure if there is supposed to be a distinction here?